_includes/posts-paginated.html

```diff
@@ -32,7 +32,7 @@
             <div class="media_image-container post-summary-image">
                 <div class="post-summary-image_container">
                     <img class="media_image"
-                         src="{{ post.thumbnail.images.fj_thumbnail.url }}"
+                         src="{{ post.thumbnail.images.full.url }}"
```

Comment:
Thanks for catching. That's should've been in another future PR. It's there because I didn't clean up my experiment properly. Removing...